tcms/xmlrpc/api/testexecution.py

```diff
@@ -223,3 +223,40 @@ def get_links(case_run_id):
     links = LinkReference.objects.filter(test_case_run=case_run_id)
     serialier = XMLRPCSerializer(links)
     return serialier.serialize_queryset()
+
+
+# workaround for keeping backward-compatibility with users of the API calling TestCaseRun.*
+@rpc_method(name='TestExecution.add_comment')
+def test_execution_add_comment(case_run_id, comment, **kwargs):
+    return add_comment(case_run_id, comment, **kwargs)
+
+
+@rpc_method(name='TestExecution.create')
+def test_execution_create(values):
```

Comment:
Missing `permissions_required` decorator, also see `test_execution_update()`. Use either one of the two variants.